recipe/meta.yaml

```diff
@@ -109,12 +116,15 @@ requirements:
     - numpy *      # [megabuild]
     - numpy        # [not megabuild]
     - pip
-    - setuptools
+    # see https://github.com/pytorch/pytorch/issues/136541
+    - setuptools <=72.1.0  # [win]
+    - setuptools  # [not win]
     - pyyaml
     - requests
     - six
     - mkl-devel {{ mkl }}   # [blas_impl == "mkl"]
     - libcblas * *_mkl      # [blas_impl == "mkl"]
+    - libblas               # [blas_impl != "mkl"]
     - libcblas              # [blas_impl != "mkl"]
     - liblapack             # [blas_impl != "mkl"]
     - libgomp   # [linux]
```

Comment:
This also applies in host obviously.

Reply:
Sorry @h-vetinari what do you mean here?

Reply:
Was talking about openmp. The good part about the code/comment you referenced in the other thread is that pytorch on windows is _the_ reason why we have anything to do with `intel-openmp` in conda-forge at all still. Once we replace this with `llvm-openmp`, we should be able to finally drop this across the board. But OK to defer this; this PR is enough work as-is.